samples/compiler_plugins/xnnpack/src/xnnpack/Conversion/StablehloToXnnpack.cpp

```diff
@@ -11,8 +11,10 @@
 #include "mlir/IR/PatternMatch.h"
 #include "mlir/IR/Verifier.h"
 #include "mlir/Parser/Parser.h"
+#include "mlir/Pass/Pass.h"
+#include "mlir/Pass/PassManager.h"
```

Comment:
nit: can you add `Pass` as a dependency here

https://github.com/ramiro050/iree/blob/ba0e5de9bccb840b1a4afe1cb8ab42808d4275e7/samples/compiler_plugins/xnnpack/BUILD.bazel#L139

and here

https://github.com/ramiro050/iree/blob/ba0e5de9bccb840b1a4afe1cb8ab42808d4275e7/samples/compiler_plugins/xnnpack/CMakeLists.txt#L110